types/aws-lambda/trigger/api-gateway-proxy.d.ts

```diff
@@ -19,7 +19,7 @@ export type APIGatewayProxyCallback = Callback<APIGatewayProxyResult>;
  * Works with HTTP API integration Payload Format version 2.0
  * @see - https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html
  */
-export type APIGatewayProxyHandlerV2 = Handler<APIGatewayProxyEventV2, APIGatewayProxyResultV2>;
+export type APIGatewayProxyHandlerV2<T = object> = Handler<APIGatewayProxyEventV2, APIGatewayProxyResultV2<T>>;
```

Comment:
I'm not sure I like the default to be nearly "allow anything".
I'd like `T=never` here and in the result so you can only return that if you specifically ask for it (with an $ExpectError test for eg. a misspelled structured result property)

Reply:
Done!
`T=never` indeed makes more sense.